src/kytos/interfaceInfo.vue

```diff
@@ -21,7 +21,50 @@
               <k-property-panel-item :name="key" :value="value" :key="key" v-if="content" v-for="(value, key) in this.metadata"></k-property-panel-item>
           </k-property-panel>
       </k-accordion-item>
-
+      <k-accordion-item title="Available Tags">
+        <div class="metadata_table">
+          <table>
+            <thead>
+              <tr>
+                <th>Type</th>
+                <th>Value Ranges</th>  
+              </tr>
+            </thead>
+            <tbody>
+              <tr v-for="(value, key) in this.available_tags">
+                <td >{{key}}</td>
+                <td >{{value}}</td>
```

Comment:
Aldo, can you try to align the text left? I have an impression that the range representation with text centered becomes a bit harder to read, for instance, check out how these squared brackets ended up:

![20231006_130140](https://github.com/kytos-ng/ui/assets/1010796/a9be5b93-1363-4c0c-834c-792856deb133)

Same comment for tag ranges `td` entries below

Reply:
Changed align to left